integration_tests/tests/alexaforbusiness.rs

```diff
@@ -0,0 +1,23 @@
+#![cfg(feature = "alexaforbusiness")]
+
+extern crate rusoto_core;
+extern crate rusoto_alexaforbusiness;
+
+use rusoto_alexaforbusiness::{AlexaForBusiness, AlexaForBusinessClient, ListSkillsRequest};
+use rusoto_core::Region;
+
+#[test]
+fn should_list_skills() {
+    let client = AlexaForBusinessClient::simple(Region::UsEast1);
+    let request = ListSkillsRequest::default();
+
+    let response = client.list_skills(&request).sync();
+    println!("response is {:#?}", response);
```

Comment:
Do we want to print this here?

Reply:
I think so! 😄 It's only displayed if the test fails or tests are run with the `--nocapture` flag.